content/client-server-api/modules/mentions.md

```diff
@@ -2,26 +2,57 @@
 ### User and room mentions
 
 This module allows users to mention other users and rooms within a room message.
-This is achieved by including a [Matrix URI](/appendices/#uris) in the HTML body of
-an [m.room.message](#mroommessage) event. This module does not have any server-specific
-behaviour to it.
+This is achieved by including metadata in the `m.mentions` content property of
+the event. The `m.mentions` property consists of an object with two fields:
```

Comment:
I wonder if we need to make it explicit that `m.mentions` is encrypted in the same way as the rest of `content`?

Reply:
I assumed that any property not given as an exception is encrypted by default?

Reply:
I hesitate because (a) there was debate about it in the MSC stage, and people might still have the wrong impression; (b) afaik the only other "mixin" property we have like this is `m.relates_to`, which is *not* encrypted.

I think a couple of words making it explicit wouldn't hurt.

Reply:
That's fair, I added a note about ti and pointed back to how clients should process push rules (for encrypted events).